erpnext_com/www/docs/user/manual/en/buying/articles/pull-items-in-purchase-order-based-on-supplier.md

```diff
@@ -1,36 +0,0 @@
-<!-- add-breadcrumbs -->
-#Pull Items in Purchase Order based on Supplier
-
-**Question:**
-
-Our Material Request has many items, each purchased from different suppliers. How to pull items from all open Material Request which are to be purchased from common Supplier?
-
-**Answer:**
-
-To pull items from Material Request for specific Supplier only, follow below given steps.
-
-####Step 1:  Default Supplier
-
-Update Default Supplier in the Item master.
-
-<img alt="Item Purchase UoM" class="screenshot" src="{{docs_base_url}}/assets/img/articles/for-supplier-2.png">
```

Comment:
also delete the images

Reply:
#413

Reply:
there are two images in the PR, delete both

Reply:
Done